.github/workflows/docs.yml

```diff
@@ -14,7 +14,16 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  Authorize:
```

Comment:
We may not need this here since this does have an on: pull_request_target trigger point. But does not hurt to have, so okay.